libs/cart/src/models/cart-coupon.ts

```diff
@@ -1,7 +1,21 @@
 import { ID } from '@daffodil/core';
 
+/**
+ * A coupon applied to the cart.
+ * Also known as a promo code.
+ */
 export interface DaffCartCoupon {
+  /**
+   * The coupon ID.
+   */
+  // TODO: change to id
```

Comment:
I tried to change this to use the DaffIdentifiable interface, but it ended up being a little tricky because coupon_id is optional. Maybe we make `coupon_id` the same as the code with Magento (which doesn't seem to have an id other than the code)?

Reply:
okay I'll remove this for now and we can address it later

Reply:
removed

Reply:
Hmm, yea the more I think about it, the more I think it makes sense. Added an issue for it here: https://github.com/graycoreio/daffodil/issues/1656